PittAPI/laundry.py

```diff
@@ -49,92 +36,85 @@ def get_status_simple(building_name):
         -> SUTH_EAST
         -> SUTH_WEST
-
-    session.hash_bits_per_character = 5
     """
 
     building_name = building_name.upper()
-    url = 'http://classic.laundryview.com/appliance_status_ajax.php?lr={}'.format(location_dict[building_name])
-    page = session.get(url)
-    soup = BeautifulSoup(page.text, 'lxml')
-
-    # What was I doing...
-    re1 = ['(\\d+)', '(\\s+)', '(of)', '(\\s+)', '(\\d+)', '(\\s+)', '((?:[a-z][a-z]+))']
-
-    rg = re.compile(''.join(re1), re.IGNORECASE | re.DOTALL)
-    search = rg.findall(str(soup))
-
-    di = {
+    url ='http://m.laundryview.com/submitFunctions.php?monitor=true&lr={}'.\
+            format(location_lookup[building_name])
+    response = requests.get(url)
+    laundry_soup = BeautifulSoup(response.text, 'lxml')
+    reFormat = re.compile(r'^([0-9]+) of ([0-9]+) available$')
+    washer_text = laundry_soup.find('span',{'id': 'washer_available'}).text
+    dryer_text = laundry_soup.find('span',{'id': 'dryer_available'}).text
+    washer_match = reFormat.match(washer_text)
+    dryer_match = reFormat.match(dryer_text)
+
+    return  {
         'building': building_name,
-        'free_washers': search[0][0],
-        'total_washers': search[0][4],
-        'free_dryers': search[1][0],
-        'total_dryers': search[1][4]
+        'free_washers': int(washer_match.group(1)),
+        'total_washers': int(washer_match.group(2)),
+        'free_dryers': int(dryer_match.group(1)),
+        'total_dryers': int(dryer_match.group(2))
     }
 
-    return di
 
 
-def get_status_detailed(building_name):
-    building_name = building_name.upper()
+def get_status_detailed(building_name, machine=None):
+    """
+    :returns: A list of washers and dryers for the passed
+              building location with their statuses
 
-    # Get a cookie
-    response = requests.get("http://www.laundryview.com/laundry_room.php?view=c&lr={}".format(location_dict[building_name]))
-    cookie = response.headers["Set-Cookie"]
-    cookie = cookie[cookie.index("=") + 1:cookie.index(";")]
+              OR
 
-    # Get the weird laundry data
-    headers = {"Cookie": "PHPSESSID={}".format(cookie)}
-    response = requests.get("http://www.laundryview.com/dynamicRoomData.php?location={}".format(
-        location_dict[building_name]), headers=headers).text
-    resp_split = response.split('&')[3:]
+              A dict of machine metadata if machine name
+              is explicity passed
 
-    cleaned_resp = []
-    for status_string in resp_split:
-        if "machine" not in status_string:
+    :param building_name: (String) one of these:
+        -> BRACKENRIDGE
+        -> HOLLAND
+        -> LOTHROP
+        -> MCCORMICK
+        -> SUTH_EAST
+        -> SUTH_WEST
+    :param machine: (String) specific machine to query
+                    eg: machine='BA02'
+    """
+    building_name = building_name.upper()
+    machines = []
+
+    url = 'http://m.laundryview.com/submitFunctions.php?monitor=true&lr={}'.\
```

Comment:
The backslash should be before the period, and the period should go on the next line with format, so it becomes `.format`